packages/connect/src/types/api/solana/index.ts

```diff
@@ -76,5 +76,6 @@ export const SolanaComposedTransaction = Type.Object({
     additionalInfo: Type.Object({
         newAccountProgramName: Type.Optional(SolanaProgramName),
         tokenAccountInfo: Type.Optional(SolanaTxTokenAccountInfo),
+        encoded_token: Type.Optional(Type.ArrayBuffer()),
```

Comment:
This one shouldn't be here, this type is the result of `solanaComposeTransaction`

Reply:
true 4d308ce31c8a7c51d2700c11a38a49bff377a958